tests/Feature/Models/CurrencyTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use App\Models\Currency;
+use App\Models\Exchange;
+use App\Models\Portfolio;
+
+beforeEach(function () {
+    $this->currency = Currency::factory()->create();
+});
+
+test('currency attributes', function () {
+    expect($this->currency)->toBeInstanceOf(Currency::class)
+        ->and($this->currency->name)->toBeString()
+        ->and(strlen($this->currency->name))->toBe(3); // ISO code assumption
```

Comment:
deze is wel heel specifiek op de factory. denk gewoon 
```suggestion
        ->and($this->currency->name)->toBeString();
```